guide/10-mapping-and-visualization-alt/part7_utility_functions_for_the_map_widget.ipynb

```diff
@@ -0,0 +1,357 @@
+{
```

Comment:
<h2>Demonstrate callback features with <code>on_click</code> and <code>on_draw_end</code> </h2><p>Our first example will take a point that a user clicks on the map, reverse geocode it from the geometry, and print out the resulting location. </p><p>Note, you can either create the GIS connection using an existing profile, or you can simply enter your username and password, e.g. gis = GIS("<a href="https://www.arcgis.com%22/" target="_blank">https://www.arcgis.com"</a>, "username", "password").</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>